build/pbm-entry.sh

```diff
@@ -4,59 +4,12 @@ set -o xtrace
 
 PBM_MONGODB_URI="mongodb://${PBM_AGENT_MONGODB_USERNAME}:${PBM_AGENT_MONGODB_PASSWORD}@localhost:${PBM_MONGODB_PORT}/?replicaSet=${PBM_MONGODB_REPLSET}"
 
-PBM_MONGO_OPTS=""
 MONGO_SSL_DIR=/etc/mongodb-ssl
 if [[ -f "${MONGO_SSL_DIR}/tls.crt" ]] && [[ -f "${MONGO_SSL_DIR}/tls.key" ]]; then
 	cat "${MONGO_SSL_DIR}/tls.key" "${MONGO_SSL_DIR}/tls.crt" >/tmp/tls.pem
-	PBM_MONGO_OPTS="--tls --tlsAllowInvalidHostnames --tlsCertificateKeyFile=/tmp/tls.pem --tlsCAFile=${MONGO_SSL_DIR}/ca.crt"
 	PBM_MONGODB_URI="${PBM_MONGODB_URI}&tls=true&tlsCertificateKeyFile=%2Ftmp%2Ftls.pem&tlsCAFile=${MONGO_SSL_DIR}%2Fca.crt&tlsInsecure=true"
 fi
 
 export PBM_MONGODB_URI
```

Comment:
do we need these lines anymore?

Reply:
Do you mean just this line 15, or some other lines as well? I'm not sure what else can be removed here, help me out. @hors wdyt?

Reply:
if you are talking about PBM_MONGODB_URI PBM agent use it https://docs.percona.com/percona-backup-mongodb/architecture.html#pbm-command-line-utility-pbm-app

Reply:
you're correct, my bad